packages/showcases/react/stories/components/indicators/VtmnRating/VtmnRating.stories.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { VtmnRating } from '@vtmn/react';
+import { Meta, Story } from '@storybook/react';
+import {
+  argTypes,
+  parameters,
+} from '@vtmn/showcase-core/csf/components/indicators/rating.csf';
+
+export default {
+  title: 'Components / Indicators / VtmnRating',
+  component: VtmnRating,
+  argTypes: {
+    ...argTypes,
+    comment: {
+      type: { name: 'number', required: false },
+      description:
+        'Comment displayed on the right side of the rating when using compact mode.',
+      defaultValue: false,
+      control: { type: 'number' },
+    },
+
```

Comment:
FYI, we just rework the `VtmnRating` component, `comments` is now available inside the component as a string property. See this PR: https://github.com/Decathlon/vitamin-web/pull/1121

I propose you globally check if all it's aligned as we did in the `VtmnRating` Svelte component here: https://github.com/Decathlon/vitamin-web/blob/main/packages/sources/svelte/src/components/indicators/VtmnRating/VtmnRating.svelte

Reply:
bf819f4 should now make the component behave the same way as the svelte one :)